docs/content/installation/nic-images/use-aws-image.md

```diff
@@ -78,12 +80,57 @@ Make sure you have an operational EKS cluster and that the namespace for your NG
         apiGroup: rbac.authorization.k8s.io
     ```
 
-3. Sign in to the AWS ECR registry that specified in the instructions on the [AWS Marketplace portal](https://aws.amazon.com/marketplace/pp/prodview-fx3faxl7zqeau?sr=0-1&ref_=beagle&applicationId=AWSMPContessa).
+1. Sign in to the AWS ECR registry that specified in the instructions on the [AWS Marketplace portal](https://aws.amazon.com/marketplace/pp/prodview-fx3faxl7zqeau?sr=0-1&ref_=beagle&applicationId=AWSMPContessa).
```

Comment:
```suggestion
3. Sign in to the AWS ECR registry that specified in the instructions on the [AWS Marketplace portal](https://aws.amazon.com/marketplace/pp/prodview-fx3faxl7zqeau?sr=0-1&ref_=beagle&applicationId=AWSMPContessa).
```

Reply:
Alan asked me to avoid using numbers, helm will figure out the correct numbers